controller/space_iterations_test.go

```diff
@@ -268,6 +268,24 @@ func (rest *TestSpaceIterationREST) TestWICountsWithIterationListBySpace() {
 	fmt.Println("iteration2 id = ", iteration2.ID)
 	assert.NotEqual(rest.T(), uuid.UUID{}, iteration2.ID)
 
+	childOfIteration2 := iteration.Iteration{
+		Name:    "Sprint 2.1",
+		SpaceID: spaceInstance.ID,
+		Path:    append(iteration2.Path, iteration2.ID),
+	}
+	iterationRepo.Create(rest.ctx, &childOfIteration2)
+	fmt.Println("iteration2 id = ", childOfIteration2.ID)
+	require.NotEqual(rest.T(), uuid.UUID{}, childOfIteration2.ID)
+
+	grnadChildOfIteration2 := iteration.Iteration{
```

Comment:
sorry. done in https://github.com/almighty/almighty-core/pull/1151/commits/19150ea07ffc361bfbcfc2cb60e7c2ca40c26c9c